doc/guides/maintaining-V8.md

```diff
@@ -382,7 +382,7 @@ git node v8 major --branch=5.1-lkgr
 
 This should be followed up with manual refloating of all relevant patches.
 
-## Proposal: using a fork repo to track upstream V8
+## Proposal: using a fork repository to track upstream V8
```

Comment:
This one is already fixed in master and can be removed to get rid of the merge conflict.

Reply:
done